apis/clusterresources/v1beta1/awsendpointserviceprincipal_types.go

```diff
@@ -60,12 +60,12 @@ type AWSEndpointServicePrincipal struct {
 	Status AWSEndpointServicePrincipalStatus `json:"status,omitempty"`
 }
 
-func (r *AWSEndpointServicePrincipal) NewPatch() client.Patch {
-	return client.MergeFrom(r.DeepCopy())
+func (principal *AWSEndpointServicePrincipal) NewPatch() client.Patch {
+	return client.MergeFrom(principal.DeepCopy())
 }
 
-func (r *AWSEndpointServicePrincipal) GetJobID(job string) string {
-	return r.Namespace + "/" + r.Name + "/" + job
+func (principal *AWSEndpointServicePrincipal) GetJobID(job string) string {
```

Comment:
Please rename receiver `principal` to `p`

Reply:
Done